### PR TITLE
Add support for new version of iTerm2

### PR DESCRIPTION
**Pull request by xream**
FYI: [https://iterm2.com/applescript.html](https://iterm2.com/applescript.html)

### CODE REVIEW

File: resources/terminal [hunk 1]
```diff
@@ -36,6 +36,28 @@ function open_iterm () {
 EOF
 }
 
+function open_iterm_new () {
```
Comment (TeckniX) [inline resources/terminal]: Can you fix the indentation of your code? I believe the standard used is 2-spaces while you have 4?

**Comment by TeckniX** [comment]
@xream I downloaded the latest version of iTerm2 and didn't run into errors while using the 'Docker CLI' button - Did you run into specific issues that required the above patch?

Thanks for the contribution btw :+1:

**Comment by xream** [comment]
@TeckniX Sorry about the indentation. I carelessly edited that part using online editor of GitHub. Maybe the nightly build of iTerm 2 changes too quickly... I think it's better to close this and wait for the stable new version of iTerm 2 :smiley:

**Comment by TeckniX** [comment]
Sounds good! Thanks for the input :)

**Comment by radnor** [comment]
I had to use this change from @xream to get iTerm2 to open correctly. I'm using version `2.9.20150720-nightly` at the moment.

**Comment by TeckniX** [comment]
I think the nightly is always problematic, as mentioned above. I have the latest stable iterm and it opens the Docker CLI without any issues.

**Pull request closed without merging**